scripts/generate_psa_constants.py

```diff
@@ -395,6 +396,8 @@ def generate_psa_constants(header_file_names, output_file_name):
 if __name__ == '__main__':
     if not os.path.isdir('programs') and os.path.isdir('../programs'):
         os.chdir('..')
+    # Allow to change the directory where psa_constant_names_generated.c is written to.
+    OUTPUT_FILE_DIR = sys.argv[1] if len(sys.argv) == 2 else "programs/psa"
```

Comment:
Please document at the begin of the file, the two ways of calling the script: without any argument or with the output file directory and the corresponding two directories where the generated file is going to be stored.

Reply:
Done in an added commit!